projects/observability/src/shared/components/topology/utils/topology-converter.ts

```diff
@@ -90,12 +91,26 @@ export class TopologyConverter {
     domRenderer: Renderer2,
     stateManager: TopologyStateManager
   ): RenderableTopologyEdge[] {
-    return edges.map(edge => {
-      const sourceNode = nodeMap.get(edge.fromNode)!;
-      const targetNode = nodeMap.get(edge.toNode)!;
+    return edges
```

Comment:
can we isolate new code with the new property?

Reply:
Done